src/petals/server/memory_cache.py

```diff
@@ -33,12 +33,10 @@ def __init__(self, device: Union[str, torch.device], max_size_bytes: Optional[in
         self._lock_metadata, self.size_decreased_event = mp.Lock(), mp.Event()
         self._current_size = mp.Value(ctypes.c_int64, 0, lock=False)
         self._handle_counter = mp.Value(ctypes.c_int64, 0, lock=False)
-        self._active_handles: Optional[Dict[Handle, TensorDescriptor]] = None
-        self._allocated_tensors: Optional[Dict[Handle, torch.Tensor]] = None
+        self._allocated_tensors: Dict[Handle, torch.Tensor] = {}
```

Comment:
Changes in this file just simplify code a bit without changing its functionality.